src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,15 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder resultStringBuilder = new StringBuilder();
+        if (value == 0) {
+            resultStringBuilder.append('0');
+        } else {
+            while (value > 0) {
+                resultStringBuilder.append(value % 2);
+                value /= 2;
+            }
+        }
+        return resultStringBuilder.reverse().toString();
```

Comment:
Do we need else block here ?

Reply:
We can without else. I added else, if the value is equal to 0, the check in the loop while was not performed.